apps/omg_watcher/lib/omg_watcher/exit_processor/standard_exit.ex

```diff
@@ -79,10 +79,13 @@ defmodule OMG.Watcher.ExitProcessor.StandardExit do
     exits_invalid_by_ife = get_invalid_exits_based_on_ifes(active_exits, tx_appendix)
     invalid_exits = active_exits |> Map.take(invalid_exit_positions) |> Enum.concat(exits_invalid_by_ife) |> Enum.uniq()
 
+    ethereum_block_time_seconds = OMG.Eth.Configuration.ethereum_block_time_seconds()
     # get exits which are still invalid and after the SLA margin
     late_invalid_exits =
       invalid_exits
-      |> Enum.filter(fn {_, %ExitInfo{eth_height: eth_height}} -> eth_height + sla_margin <= eth_height_now end)
+      |> Enum.filter(fn {_, %ExitInfo{eth_height: eth_height}} ->
```

Comment:
We're moving away from single pipes. Could you update `invalid_exits |> Enum.filter(fn...` to `Enum.filter(invalid_exits, fn...`? Thanks!